src/main/java/com/vinylteam/vinyl/web/handler/ApiExceptionHandler.java

```diff
@@ -33,13 +34,12 @@
     public ResponseEntity<Object> handleServerExceptionApiExceptionHandler(ServerException exception) {
```

Comment:
Could you please remove blank lines between handleForbiddenExceptionApiExceptionHandler and handleServerExceptionApiExceptionHandler ?